src/eureka/S4_generate_lightcurves/plots_s4.py

```diff
@@ -201,7 +201,7 @@ def lc_driftcorr(meta, wave_1d, optspec, optmask=None):
         plt.xlabel(r'Wavelength ($\mu m$)')
         plt.colorbar(label='Normalized Flux')
 
-        if len(meta.wave_low) > 1:
+        if len(meta.wave) > 1 and len(wave_1d) != meta.nspecchan:
```

Comment:
Did you mean to change the first part from `wave_low` to `wave`?

Reply:
Yeah, the length of both arrays must be the same, and `meta.wave` is shorter to write than `meta.wave_low`